esp/scm/ws_dfu.ecm

```diff
@@ -871,44 +879,72 @@ ESPStruct DFUFilePart
 
 ESPStruct DFUFileAccessInfo
 {
-    string MetaInfoBlob;
-    string ExpiryTime;
-    // {NumParts, FileParts} depend on ReturnFileInfo in request
-    int NumParts;        // number of parts in logical file
-    ESParray<EspStruct DFUPartLocation> FileLocations;
-    ESParray<EspStruct DFUFilePart> FileParts;
+    string MetaInfoBlob;                               // Contains serialized from of all meta data required + signature if security enabled
+    string ExpiryTime;                                 // After this time, the meta information cannot be used to communicate with dafilesrv's 
+    int NumParts;                                      // number of parts in logical file
+    ESParray<EspStruct DFUPartLocation> FileLocations; // List of hosts of parts  
```

Comment:
Can the row service port change? If so could we add the row service port number to DFUPartLocation?

Reply:
It can't vary for any given single request.
So the single 'fileAccessPort' field in this response should suffice.